libs/features/personalization/personalization.js

```diff
@@ -1034,6 +1034,36 @@ export const combineMepSources = async (persEnabled, promoEnabled, mepParam) =>
   return persManifests;
 };
 
+const preloadMarqueeImages = () => {
+  const isDesktop = window.innerWidth >= 1200;
+  const isTablet = window.innerWidth >= 600 && !isDesktop;
+  const isMobile = window.innerWidth < 600;
+  const headerBlock = document.querySelector('main > div:first-child > div'); // *marquee or text
+  const nonBgImages = headerBlock?.querySelectorAll(':scope > div:not(:first-child) img');
+  const hasBackground = headerBlock?.querySelectorAll(':scope > div')?.length > 1;
```

Comment:
Question: should you first figure out hasBackground?  If it does, nonBg images is the selector you have.  If it's not, I would think it would be all images in the block.